api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3629 @@
+{
```

Comment:
<p>Note in the first point : trs should be TRS</p><p> </p><p>Last point</p><p>"plot the following figure plots" doesn't seem right</p><p> </p><p>"what im going to see" I'm not sure what this is saying or what "im" is meant to be.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>